pom.client.xml

```diff
@@ -428,6 +428,10 @@
               <artifactId>codesnippet-doclet</artifactId>
               <version>${codesnippet4javadoc.version}</version>
             </docletArtifact>
+            <additionalOptions>
+              <additionalOption>-maxLineLength 120</additionalOption>
+              <additionalOption>-snippetpath ${project.basedir}/src/samples/java</additionalOption>
+            </additionalOptions>
             <additionalOptions>-maxLineLength 120 -snippetpath ${project.basedir}/src/samples/java</additionalOptions>
```

Comment:
Is this specified twice here? the additionalOptions are also defined below.

Reply:
the one below was already there. But I think i was doing nothing. Anywhere I left it there. Not sure if it depends on maven version what declaration would work

Reply:
@JonathanGiles I do not know the maven magic. Will this overwrite the set that Victor just added?

Reply:
Duplicating is not the right thing to do here. The right thing would be to determine which is correct given our current tooling. From [here](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#additionalOptions), it seems like the new approach is more appropriate and the previously approach is probably the old way, although I can't tell for sure. Having said that, I don't think this change relates to this PR, so it should be moved out to a separate pull request so that this one may focus on the line length issue for source code.